src/periphery/MimRewardsDistributor.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "BoringSolidity/BoringOwnable.sol";
+import "BoringSolidity/ERC20.sol";
+import "BoringSolidity/libraries/BoringERC20.sol";
+
+/// @notice basic simple mim reward distributor
+/// To be improved to support per-address mim distribution allocations
+contract MimRewardDistributor is BoringOwnable {
```

Comment:
this does not work yet

Reply:
Yes, I'm making a working version to repayment